pcweb/components/webpage/navbar/inkeep.py

```diff
@@ -1,23 +1,23 @@
 """UI and logic inkeep chat component."""
 
-from typing import Set
 import reflex as rx
 from reflex.event import EventHandler
-from reflex.vars import ImportVar, Var
+from reflex.vars import ImportVar
+from reflex.vars import Var
 
 
 class Search(rx.Component):
     tag = "SearchBar"
 
-    special_props: Set[Var] = {
-        Var.create_safe("{...searchBarProps}", _var_is_string=False)
+    special_props: set[Var] = {
+        Var.create_safe("{...searchBarProps}", _var_is_string=False),
     }
 
     is_open: Var[bool] = False
 
-    on_close: EventHandler[lambda: []]
+    on_close: EventHandler[list]
```

Comment:
I think we want to keep the original way as it's the explicit way to define specs

Reply:
Fixed! In the new PR that i'll open!